shell/src/main/java/alluxio/cli/fs/command/PersistCommand.java

```diff
@@ -34,19 +43,40 @@
  */
 @ThreadSafe
 public final class PersistCommand extends AbstractFileSystemCommand {
+  private static final Option PARALLELISM_OPTION =
+      Option.builder()
+          .longOpt("parallelism")
+          .argName("# threads")
+          .numberOfArgs(1)
```

Comment:
would it make sense to set a higher default, e.g. 4 or 8?

Reply:
Good idea, I've changed it to 4. Has a side effect of improving our test run time...